dev/library/ctrl.js

```diff
@@ -69,6 +69,7 @@
     //Preview popup
     vm.showPreview = function (video) {
       var ext = video.attributes.external_file_name.split('.').pop();
+      ext = (ext === 'wmv') ? 'webm' : ext;
```

Comment:
Why?

Reply:
Does it mean that we can't play wmv?

Reply:
every browser does not support WMV as a type of file. Aside from IE, this type of format is unlikely to be compatible with most browsers.

Reply:
So how changing 'wmv' to 'webm'  helps us?